spa/src/services/subscription-manager/index.ts

```diff
@@ -2,12 +2,31 @@ import Api from "../../api";
 import { AxiosError } from "axios";
 import { reportError } from "../../utils";
 import { GHSubscriptions } from "../../../../src/rest-interfaces";
+import { RestSyncReqBody } from "~/src/rest-interfaces";
+
+async function syncSubscription(subscriptionId:number, reqBody: RestSyncReqBody): Promise<GHSubscriptions | AxiosError> {
+	try {
+		const response= await Api.subscriptions.syncSubscriptions(subscriptionId,reqBody);
+		const isSuccessful = response.status === 202;
+		if(!isSuccessful) {
+			reportError(
+				{ message: "Response status for backfilling subscriptions is not 202", status: response.status },
+				{ path: "syncSubscription" }
+			);
+		}
+
+		return response.data;
```

Comment:
I don't think the response from this endpoint returns any `data`, so this might error out.